### PR TITLE
Refactoring and clean up #2

### PR DESCRIPTION
**Pull request by drowe67**
1. Removing dead code, such as old, unused experimental features. :heavy_check_mark: 
2. Moving code out of the too-large `main.cpp` into other files.  Especially code that is rarely modified.  `main.cpp` has been reduced from 5000 to 3000 lines, most of that is the real time code that is tightly coupled. :heavy_check_mark: 
3. Disable or make optional the many `fprintf(stderr,...)` to the console. :heavy_check_mark:

### CODE REVIEW

**Comment by drowe67** [comment]
@tmiw :point_up:

**Comment by drowe67** [comment]
@tmiw is it OK with you if I merge this?  I don't want to mess up any of your PRs that are in progress.  Nothing has changed functionally, just code moved around to different files.  I'm pretty sure Git will merge everything OK.

File: src/main.cpp [hunk 1]
```diff
@@ -873,13 +788,10 @@ MainFrame::~MainFrame()
...
     }
 
-    //m_togRxID->Disconnect(wxEVT_UPDATE_UI, wxUpdateUIEventHandler(MainFrame::OnTogBtnRxIDUI), NULL, this);
-    //m_togTxID->Disconnect(wxEVT_UPDATE_UI, wxUpdateUIEventHandler(MainFrame::OnTogBtnTxIDUI), NULL, this);
     m_togBtnOnOff->Disconnect(wxEVT_UPDATE_UI, wxUpdateUIEventHandler(MainFrame::OnTogBtnOnOffUI), NULL, this);
     m_togBtnSplit->Disconnect(wxEVT_UPDATE_UI, wxUpdateUIEventHandler(MainFrame::OnTogBtnSplitClickUI), NULL, this);
     m_togBtnAnalog->Disconnect(wxEVT_UPDATE_UI, wxUpdateUIEventHandler(MainFrame::OnTogBtnAnalogClickUI), NULL, this);
-    //m_togBtnALC->Disconnect(wxEVT_UPDATE_UI, wxUpdateUIEventHandler(MainFrame::OnTogBtnALCClickUI), NULL, this);
-    //m_btnTogPTT->Disconnect(wxEVT_UPDATE_UI, wxUpdateUIEventHandler(MainFrame::OnTogBtnPTT_UI), NULL, this);
+     //m_btnTogPTT->Disconnect(wxEVT_UPDATE_UI, wxUpdateUIEventHandler(MainFrame::OnTogBtnPTT_UI), NULL, this);
```
Comment (tmiw) [inline src/main.cpp:794]: Should the commented line just be removed since PTT connect/disconnect is already being handled elsewhere?

**Review by tmiw — COMMENTED** [review]
Looks good other than one minor thing.

**Pull request merged**